processors/config/rawSvtAnaHits_cfg.py

```diff
@@ -0,0 +1,73 @@
+import HpstrConf
+import sys,os
+import baseConfig as base
+
+options = base.parser.parse_args()
+
+# Use the input file to set the output file name
+root1_file = options.inFilename[0]
+root2_file = options.outFilename
+
+print('Root file Input: %s' % root1_file)
+print('Root file Output: %s' % root2_file)
+
+p = HpstrConf.Process()
+
+p.run_mode = 1
+p.skip_events = options.skip_events
+if(options.nevents>-1):
+    p.max_events = options.skip_events+options.nevents
+p.max_events = 200000
+
+# Library containing processors
+p.add_library("libprocessors")
+
+###############################
+#          Processors         #
+###############################
+
+rawAnaSvt = HpstrConf.Processor('svtana','SvtRawDataAnaProcessor')
+
+###############################
+#   Processor Configuration   #
+###############################
+
+#SvtRawAnaData
+
+rawAnaSvt.parameters["debug"] = 0
+rawAnaSvt.parameters["anaName"] = 'rawSvtHitAna'
+rawAnaSvt.parameters["trkrHitColl"] = 'SVTRawHitsOnTrack_KF'#'SVTRawTrackerHits'
+rawAnaSvt.parameters["histCfg"] = os.environ['HPSTR_BASE']+'/analysis/plotconfigs/svt/rawSvtAnaHits.json'
+rawAnaSvt.parameters["sample"] = 0 
+
+RegionPath = os.environ['HPSTR_BASE']+"/analysis/selections/svtHit/"
+
+rawAnaSvt.parameters["baselineFile"]="/sdf/group/hps/users/rodwyer1/hps_14552_offline_baselines.dat"
```

Comment:
These shouldn't be hard coded paths to your user dir on sdf. We should add these dat files to hpstr and you should use the path relative to HPSTR_BASE.

Reply:
Done :)